oml/registry/models.py

```diff
@@ -10,11 +12,28 @@
     "resnet": ResnetExtractor,
     "vit": ViTExtractor,
     "vit_clip": ViTCLIPExtractor,
+    "extractor_with_mlp": ExtractorWithMLP,
 }
 
 
+def raise_if_needed(extractor_cfg: TCfg, kwargs: Dict[str, Any], model_name: str) -> None:
+    if extractor_cfg.get("weights", None) and kwargs.get("weights", None):
+        raise ValueError("You should only provide one weight for extractor_with_mlp.")
+    elif extractor_cfg.get("weights", 0) is None and kwargs.get("weights", None):
```

Comment:
why zero is here `("weights", 0)`?

Reply:
Because of `is None`

Reply:
`bool(extractor_cfg.get("weights", True)`

Reply:
```
None -> False
0 -> ""
```

Reply:
please, add test that we raise an error when it's needed